src/utils/get-options/index.js

```diff
@@ -0,0 +1,4 @@
+import getSSLOptions from './ssl';
```

Comment:
IMO ` src/utils/get-options` is too complex name for only `getVideoOptions` and `getSSLOptions` could we make it little bit more specific?

Reply:
I discussed it with @VasilyStrelyaev and we decided that `get-options` is good enough for a directory.